spec/models/metric/ci_mixin/capture_spec.rb

```diff
@@ -86,6 +86,8 @@ def verify_perf_capture_queue(last_perf_capture_on, total_queue_items)
       Timecop.freeze do
         Timecop.travel(Time.now.end_of_day - 6.hours)
         verify_perf_capture_queue(nil, 1)
+        Timecop.travel(Time.now + 20.minutes)
+        verify_perf_capture_queue(nil, 1)
```

Comment:
I added this so that it tests that no additional messages are added to the queue calling realtime a second time when `last_perf_capture_on` ids nil.